using_images/xpaas_images/a_mq.adoc

```diff
@@ -5,4 +5,92 @@
+
+. In a terminal, change into the root directory of your OpenShift master. It should contain the 'examples' subdirectory.
+. Create the image stream from which the A-MQ image will be obtained:
++
+----
+# oc create -f examples/application-templates/jboss-image-streams.json -n <project>
+----
+. Create the S2I templates for your A-MQ application:
++
+----
+# oc create -f examples/application-templates/amq/amq6.json -n <project>
+# oc create -f examples/application-templates/amq/amq6-persistence.json -n <project>
+----
+
+You will now have the JBoss A-MQ xPaaS images stream and S2I application templates available for use in your OpenShift project. 
+
+To create an application based on the template, follow instructions in the *Creating a Configuration From a Template* section of the link:../dev_guide/templates.html[Creating New Applications] topic in the OpenShift documentation. For a reference on template parameters specified when creating the application, see the xref:configuring-params[Application Template Parameters] section below.
+
+== Configuring the JBoss A-MQ Image
+
+[[configuring-params]]
+=== Application Template Parameters
+
+Basic configuration of the JBoss A-MQ xPaaS image is performed by specifying values of application template parameters. The following parameters can be configured:
+
+AMQ_RELEASE::
+  The JBoss A-MQ release version. This determines which JBoss A-MQ image will be used as a basis for the application. At the moment, only version _6.2_ is available.
+APPLICATION_NAME::
+  The name of the application used internally in OpenShift. It is used in names of services, pods and other objects within the application.
+MQ_USERNAME::
+  The user name used for authentication to the broker. In a standard non-containerized JBoss A-MQ, you would specify the user name in the 'AMQ_HOME/opt/user.properties' file. If no value is specified, a random user name is generated.
+MQ_PASSWORD::
+  The password used for authentication to the broker. In a standard non-containerized JBoss A-MQ, you would specify the password in the 'AMQ_HOME/opt/user.properties' file. If no value is specified, a random password is generated.
+MQ_PROTOCOL::
+  Comma-separated list of the messaging protocols used by the broker. Available options are _amqp_, _mqtt_, _openwire_, and _stomp_. If left empty, all available protocols will be available. Please note that for integration of the image with Red Hat JBoss Enterprise Application Platform, the _openwire_ protocol must be specified, while other protocols can be optionally specified as well.
+MQ_QUEUES::
+  Comma-separated list of queues available by default on the broker on its startup.
+MQ_TOPICS::
+  Comma-separated list of topics available by default on the broker on its startup.
+
+[[configuring-sti]]
+=== Configuration Using STI
+
+Configuration of the JBoss A-MQ image can also be modified using the Source-to-image feature, described in full detail at link:../s2i_images/overview.html[Source-to-image (S2I)].
```

Comment:
I would suggest linking here to either `../../creating_images/s2i.html` or `../../architecture/core_concepts/builds_and_image_streams.adoc#source-build`, which have more information about S2I.